app/views/articles/_form.html.erb

```diff
@@ -31,7 +31,7 @@
     </div>
 
     <div class="field">
-      <%= f.label :tag_list, "Tags" %>
+      <%= f.label :tag_list, 'Tags: allowed characters include [ a-z, 0-9, - ]' %>
```

Comment:
Oh yeah, I mentioned that I was going to break up a lot of stuff into partials in a previous PR. I know it's super messy right now :/